extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -84,10 +85,16 @@ const QueryInfo = (queryResult: RemoteQueryResult) => (
         {queryResult.queryFileName}
       </a>
     </span>
-    <span>
+    <span className="vscode-codeql__query-file">
       <CodeSquareIcon size={16} />
       <a className="vscode-codeql__query-file-link" href="#" onClick={() => openQueryTextVirtualFile(queryResult)}>
-        query
+        Query
+      </a>
+    </span>
+    <span>
+      <TerminalIcon size={16} />
+      <a className="vscode-codeql__query-file-link" href={queryResult.workflowRunUrl}>
+        Logs
```

Comment:
I notice that the link goes around the text. This means the icon itself is not clickable. Can you move the icon inside the link? This would be the same for the `Query` link above and the `{queryResult.queryFileName}` above that.

Reply:
Good idea! Done in https://github.com/github/vscode-codeql/pull/1165/commits/a0e27f01bfc1b2c5cfaae325627fe508b4af1ffa. Had to tweak the CSS a bit, but I think it's all good now: 

![image](https://user-images.githubusercontent.com/42641846/155701585-a7f94e07-0855-49a0-8a6d-b8d38acd2e33.png)